demisto_sdk/commands/lint/linter.py

```diff
@@ -86,6 +96,7 @@ def __init__(
         req_2: list,
         docker_engine: bool,
         docker_timeout: int,
+        docker_image: str = "from-yml",
```

Comment:
can we use DockerImageFlagOption.FROM_YML ? or does that cause circle deps issues?

Reply:
Yes - I moved it outside of the liter class.